app/models/payoff.rb

```diff
@@ -0,0 +1,16 @@
+class Payoff < ApplicationRecord
+  belongs_to :loan
+  has_many :received_payments
+  has_one :deal, through: :loan
+  has_many :line_items, dependent: :destroy
+  accepts_nested_attributes_for :line_items
```

Comment:
always order this stuff in the same order in every file.  Personally I would put the nested attributes at the bottom.